test/integration-config-centralledger.json

```diff
@@ -1,6 +1,23 @@
 {
   "PORT": 3001,
   "HOSTNAME": "http://central-ledger",
+  "DATABASE": {
+    "DIALECT": "mysql",
+    "HOST": "db-int",
+    "PORT": 3306,
+    "USER": "central_ledger",
+    "PASSWORD": "cVq8iFqaLuHy8jjKuA",
```

Comment:
How about changing this string to default "password" string that we use for dev?